MarsOnboardingTask/Features/Languages.feature

```diff
@@ -0,0 +1,26 @@
+Feature: This test suite runs language related test scenarios
+
+ 
+
+Scenario: 1. Verify user is able to add languages with language level
+Given User logs into Mars application
+When User adds 'French' language with 'Basic' language level into the languages list
+Then The 'French' language with 'Basic' language level should be added to languages list
+
+Scenario: 2. Verify User is able to update an existing language with a new language level
+Given User logs into Mars application
+When User updates the 'French' language with 'Fluent' language level from the languages list
+Then The language level for 'French' language should be updated to 'Fluent'
+
+Scenario: 3. Verify user is able to delete an existing known language
+When The User deletes the language 'French'
+Then The language 'French' should not be visible on the profile page
```

Comment:
Please use tables  and pass values to your scenarios for example look at the below scenario 
This way you will get exposure to use them  and will know how to pass test data to the tests


Example 

	Given I logged into the portal
	When I click on tab Education
	And I add my education 
	Then I am able to see my education'

Would  change to 


@Education
Scenario Outline: 1-Add education
	Given I logged into the portal
	When I click on tab Education
	And I add my education including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'
	Then I am able to see my education details including '<Country>', '<University>', '<Title>', '<Degree>', '<Graduation Year>'

	Examples:
	| Country     | University                      | Title  | Degree  | Graduation Year |
	| New Zealand | Whitireia Community Polytechnic | M.Tech | Diploma | 2021            |
	| Venezuela   | Hhutech University              | B.A    | Diploma | 2011            |

See  https://cucumber.io/docs/gherkin/reference/

Reply:
updated as per the review comments